Document_in_Progress.md

```diff
@@ -0,0 +1,87 @@
+
+Together with the DSRU, Ackama went on a journey from using open source technology for reasons of price and flexibility to building an entirely open source ecosystem whereby the website, apps, data and scripts are all open source technical components which work together to provide the NZSL dictionary. The community has access to all of these components and can improve upon them or add new features. By open sourcing the NZSL dictionary we encourage the wider community to be involved in making NZSL more accessible. We are also enabling international reuse of the codebase for other countries to have a technical head start in creating their own online sign language dictionary.
+
+In May 2018 the NZSL Android app got up to the number 6 ranking in the worldwide category “Top Free in Books and Reference” apps on Google Play.
+
+New Zealand Sign Language Dictionary consists of 3 major units
+* Ruby on Rails Website (this repo)
+* Mobile apps <add address>
+* Freelex, (add address) an open source project for maintaining the signs lexicon data.
+
+[![Build Status](https://travis-ci.org/ODNZSL/nzsl-online.svg?branch=master)](https://travis-ci.org/ODNZSL/nzsl-online)
+[![Code Climate](https://codeclimate.com/github/ODNZSL/nzsl-online/badges/gpa.svg)](https://codeclimate.com/github/ODNZSL/nzsl-online)
+[![Test Coverage](https://codeclimate.com/github/ODNZSL/nzsl-online/badges/coverage.svg)](https://codeclimate.com/github/ODNZSL/nzsl-online/coverage)
+[![Issue Count](https://codeclimate.com/github/ODNZSL/nzsl-online/badges/issue_count.svg)](https://codeclimate.com/github/ODNZSL/nzsl-online)
+
+## The Set Up for you wonderful open source contributors
+
+Please read carefully the CONTRIBUTORS.md (add link?) before you follow the setup steps.
+
+Fork the official repo [ODNZSL repo](https://github.com/ODNZSL/nzsl-online), so you have a copy.
+Clone your copy
+
+```
+git clone <your copy>
+cp env-example .env
+bundle
+yarn
+bundle exec rails s
+```
+A suggestion is to create a local staging branch ( You can really call it what you want ) that acts as your local master. Then branch from your staging/master branch when resolving issues or adding features.
+
+  * Create your feature branch. For example `NZSL-##Bugfix/Back-to-search-results`
+  * Make changes
+  * Commit you changes
+  * Push to the base staging branch
+  * Create a new pull request
+
+## Issues (user stories) that require design and a design process
+
+a high level design story outcome from design team
+
+## Brand guidelines
+
+## Tech requirements ??
+
+Ruby on Rails, HAML, BEM
+
+## Release History ??
+
+## Meta ??
+
+## Deployment
+
+NZSL has taken ownership of the project at their [ODNZSL repo](https://github.com/ODNZSL/nzsl-online).
+
+Open source contributors pull requests should be to the ODNZSL Staging branch. The admin for ODNZSL is currently [Elspeth Dick](elspeth@rabidtech.co.nz).
```

Comment:
This will probably need to feature @Br3nda as well and we might need to list an ops account here or something for Ackama